tests/helpers/testhelpers.py

```diff
@@ -10,10 +11,22 @@
 TEST_CASE_DIR = pathlib.Path(__file__).parent.parent / "test_case_files"
 
 VALID_CASES_BASEDIR = TEST_CASE_DIR / "valid"
+INVERSE_CASES_BASEDIR = TEST_CASE_DIR / "inverse"
+
 INVALID_CASES_BASEDIR = TEST_CASE_DIR / "invalid"
+
 RESOURCES_BASEDIR = pathlib.Path(__file__).parent.parent / "resources"
 
 
+class DIR(enum.Enum):
+    """Used as arguments to define what test case directory to look for files
+    in
+    """
+
+    VALID = VALID_CASES_BASEDIR
+    INVERSE = INVERSE_CASES_BASEDIR
+
+
```

Comment:
```suggestion
```
I would advise against this approach, it creates a severe amount of duplication in the test resources. Instead of doing this, you can just add a `stripped.txt` file in each of the valid case directories.

Create a small wrapper namedtuple for the test cases, something like `ValidTestCase("ValidTestCase", "input output stripped_output")`, and use that for the parameterized tests instead.